Add bluechi is online tool (#964)
engelmi/hirte: doc/man/bluechi-is-online.1.md

% bluechi-is-online 1

## NAME

> bluechi-is-online - Command line tool to check and monitor the connection state of BlueChi's core components.

Reviewer: ```suggestion
bluechi-is-online - Command line tool to check and monitor the connection state of Eclipse BlueChi™'s core components.
```

Reviewer: Do all the consequent occurrences should be `Eclipse BlueChi™` as well?

Author: Its sufficient to have it once in the description (as we do in the other man pages). Added the basic description with the full name and TM symbol.

## SYNOPSIS

**bluechi-is-online** [*system|node|agent*] [*options*]

## DESCRIPTION

Command line tool to check and monitor the connection state of BlueChi's core components. It uses the public D-Bus API provided by `bluechi-controller` and/or `bluechi-agent`.

**bluechi-is-online [OPTIONS]**

## OPTIONS

#### **--help**, **-h**

Print usage statement and exit.

#### **--version**, **-v**

Print current bluechictl version

## Commands

> ### **bluechi-is-online** [*system*] [*options*]

Reviewer: ```suggestion
### **bluechi-is-online** *system* [*options*]
```
The command method is not optional and not a parameter, so it shouldn't be in brackets. Same in other command examples

Author: Done.

Checks the connection state of `bluechi-controller` and all expected `bluechi-agent`s. Runs on primary node.
Uses the D-Bus property `Status` of the `org.eclipse.bluechi.Controller` interface.
If the `Status` is `up`, exit code is 0. Otherwise 1.

**Options:**

**--wait**

> Initial time to wait till the expected system status is `up`. In milliseconds.

Reviewer: ```suggestion
Initial time in milliseconds to wait until the expected system status is `up`.
```
Same in the other occurrences of `--wait` description

Author: Done.

Reviewer: @engelmi also `until` instead of `till`, "till" is more a spoken form

Author: Done.
